source/documentation/github/access-github-alphagov.md

```diff
@@ -0,0 +1,16 @@
+## Access GitHub Alphagov
+
+People joining GDS do not automatically have access to [Alphagov][] on [GitHub][]. To grant access to Alphagov, user needs to [create a new][] or share an existing GitHub username with the team. All users are required to enable [two-form authentication][] on their GitHub accounts.
+
+
+It’s the individual teams’ responsibility to grant that user access to relevant Alphagov [teams][]. 
+
+Once setup is completed, user can access Alphagov resources using the GitHub portal or [SSH connection] (https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+
+
+[GitHub]: https://www.github.com/
+[Alphagov]: https://www.github.com/alphagov/
+[create a new]: https://github.com/join
+[two-form authentication]: https://help.github.com/en/github/authenticating-to-github/configuring-two-factor-authentication
+[teams]: https://github.com/orgs/alphagov/teams
+[SSH connection]: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
```

Comment:
```suggestion
[GitHub]: https://www.github.com/
[alphagov]: https://www.github.com/alphagov/
[two-factor authentication]: https://help.github.com/en/github/authenticating-to-github/configuring-two-factor-authentication
[teams]: https://github.com/orgs/alphagov/teams
```

Reply:
@issyl0 the `[GitHub]` and `[two-factor authentication]` references are unused, did you intend to link to them above?

Reply:
👀 Thanks, I edited my suggestions (in this awful UI)!